README.md

```diff
@@ -52,6 +52,7 @@ To build opencl-clang as a standalone project, you need to obtain pre-built LLVM
 and SPIR-V Translator libraries. **Note:** currently this kind of build is
 supported on Linux only.
 
+opencl-clang is used as part of another CMake project, you will need to define `OPENCL_CLANG_BUILD_EXTERNAL`.
```

Comment:
suggest to add a new empty line after line 55

Reply:
done, thanks.